functions/clients/rawg.go

```diff
@@ -22,5 +23,6 @@ func NewRawg(rawgToken string) *Rawg {
 }
 
 func (r *Rawg) GetGame(ctx context.Context, game string) (*rawg.GameDetailed, error) {
-	return r.client.GetGame(ctx, game)
+	detail, err := r.client.GetGame(ctx, game)
+	return detail, fmt.Errorf("rawg.go: %v", err)
```

Comment:
Leave this as it was. Returning errorf will prevent it from being converted into a more specific error. If this function gets more complicated later we can do this with ErrorW (I think)